ceph/rados/core_workflows.py

```diff
@@ -4615,3 +4615,71 @@ def get_rados_df(self, pool_name: str = None):
         out = self.run_ceph_command(cmd=_cmd, client_exec=True)
 
         return out["pools"][0] if pool_name else out
+
+    def get_service_list(self, service_type=None):
```

Comment:
Please check existing method `def list_orch_services(self, service_type=None) -> list`

Reply:
Updated the code.